src/pages/studyView/styles.module.scss

```diff
@@ -18,4 +18,14 @@
 
 .studyViewSurvivalPlotBody {
     position: relative
+}
+
+.studyViewChartContainer {
+    border: 1px solid #d3d3d3
+}
+
+.studyViewGridItem {
+    .react-grid-item:hover {
```

Comment:
fyi, i don't think nesting of classes will work the way you might expect in css modules.   react-grid-item will also get "uniqueified" and need to be applied in js.  it will not apply contextually.   if i'm right, you may want to use :global() to achieve contextual application.